features/step_definitions/stamp_duty.rb

```diff
@@ -62,10 +62,6 @@
   @stamp_duty.select('buying my next home', from: @buyer_type)
 end
 
-Given("I am an additional or buy-to-let property buyer") do
-  @stamp_duty.select('buying an additional or buy-to-let property', from: @buyer_type)
-end
```

Comment:
Ha, my bad, looks like this is still needed for the Scottish version of the tool: the test fails elsewhere without it. If you restore this I think all the tests run fine.